runtime/errors.go

```diff
@@ -216,3 +217,29 @@ func (e *TransactionParameterTypeNotStorableError) Error() string {
 		e.Type.QualifiedString(),
 	)
 }
+
+// ExtendedParsingCheckingError is a special error which aids in debugging checking problems
+// by providing extra information about the state of the environment
+// Separate package to prevent cyclic imports with checker tests
+type ExtendedParsingCheckingError struct {
+	Err            error
+	RuntimeStorage *InterpreterRuntimeStorage
+	Functions      stdlib.StandardLibraryFunctions
```

Comment:
How are these used for debugging? The fields in `interpreterRuntimeStorage` are also not exported, so is exposing the storage here (and exporting `interpreterRuntimeStorage` as `InterpreterRuntimeStorage` below) useful?

Reply:
> How are these used for debugging?

They are dumped to a file for further usage. I didn't analyze what was particularly useful, just grabbed all there is

Reply:
Given that the runtime storage value doesn't provide any info, OK if I remove it?

Also, the functions that are injected are always the same, so IDK how useful it it is to include them in the error.

Reply:
> Given that the runtime storage value doesn't provide any info, OK if I remove it?
Feel free if you think it doesn't provide any value, and add what you think it's best - you are the best person to know what internal elements might be helpful